ai/ai.go

```diff
@@ -98,9 +99,10 @@ func NewAIPlayer(game *cs.Game, techStore *cs.TechStore, player *cs.Player, play
 				cs.ShipDesignPurposeBomber:                "Bomber",
 				cs.ShipDesignPurposeStructureBomber:       "Structure Bomber",
 				cs.ShipDesignPurposeSmartBomber:           "Smart Bomber",
-				cs.ShipDesignPurposeFighter:               "Stalwart Defender",
+				cs.ShipDesignPurposeStartingFighter:       "Stalwart Defender",
```

Comment:
Need to check if the AI ends up naming its starting fleets all "stalwart defender"